crates/bevy_utils/macros/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "bevy_utils_macros"
+version = "0.9.0"
+edition = "2021"
+description = "Derive implementations for bevy_utils"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+keywords = ["bevy"]
+
+[lib]
+proc-macro = true
+
+[dependencies]
+syn = { version = "1.0", features = ["full", "parsing", "extra-traits"] }
+quote = "1.0"
+bevy_macro_utils = { version = "0.9.0", path = "../../bevy_macro_utils" }
```

Comment:
This is really confusing to now have both `bevy_macro_utils` and `bevy_utils_macros`.

Reply:
Already thought about that on discord. But no answer from Alice yet